flink-connector-hbase-2.2/pom.xml

```diff
@@ -152,6 +148,18 @@ under the License.
 					<groupId>org.apache.hadoop</groupId>
 					<artifactId>hadoop-hdfs</artifactId>
 				</exclusion>
+				<exclusion>
+					<groupId>org.apache.commons</groupId>
+					<artifactId>commons-lang3</artifactId>
+				</exclusion>
+				<exclusion>
+					<groupId>commons-io</groupId>
+					<artifactId>commons-io</artifactId>
+				</exclusion>
+				<exclusion>
+					<groupId>org.jruby.jcodings</groupId>
+					<artifactId>jcodings</artifactId>
+				</exclusion>
```

Comment:
I'm assuming we don't know why these are excluded?

Reply:
Both are excluded to achieve dependency convergence, but I reworked this part a bit now.